.npmrc

```diff
@@ -0,0 +1 @@
+strict-peer-dependencies=false
```

Comment:
Why is this necessary, should we be using resolutions instead?

Reply:
I believe yarn defaults to false while pnpm defaults to true so it needs to be set to use rules_js.

If using resolutions isn't required then I'm not sure if it should be done in this PR?